### PR TITLE
docs: update quickstart with OpenLLM

### PR DESCRIPTION
**Pull request by aarnphm**
## What does this PR address?

Fixes #(issue)

## Before submitting:

- [ ] Does the Pull Request follow [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary) naming? Here are [GitHub's
      guide](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) on how to create a pull request.
- [ ] Does the code follow BentoML's code style, `pre-commit run -a` script has passed ([instructions](https://github.com/bentoml/BentoML/blob/main/DEVELOPMENT.md#style-check-auto-formatting-type-checking))?
- [ ] Did you read through [contribution guidelines](https://github.com/bentoml/BentoML/blob/main/CONTRIBUTING.md#ways-to-contribute) and follow [development guidelines](https://github.com/bentoml/BentoML/blob/main/DEVELOPMENT.md#start-developing)?
- [ ] Did your changes require updates to the documentation? Have you updated
      those accordingly? Here are [documentation guidelines](https://github.com/bentoml/BentoML/tree/main/docs) and [tips on writting docs](https://github.com/bentoml/BentoML/tree/main/docs#writing-documentation).
- [ ] Did you write tests to cover your changes?

### CODE REVIEW

File: docs/source/quickstarts/deploy-a-large-language-model-with-openllm-and-bentoml.rst [hunk 1]
```diff
@@ -85,30 +122,51 @@ The server is now active at `http://0.0.0.0:3000 <http://0.0.0.0:3000/>`_. You c
...
         .. code-block:: bash
 
-         curl -X 'POST' \
-            'http://0.0.0.0:3000/prompt' \
-            -H 'accept: text/plain' \
-            -H 'Content-Type: text/plain' \
-            -d '$PROMPT' # Replace $PROMPT here with your prompt.
+
+           curl -X 'POST' \
+               'http://0.0.0.0:3000/v1/generate' \
```
Comment (Sherlock113) [inline docs/source/quickstarts/deploy-a-large-language-model-with-openllm-and-bentoml.rst:131]: Shouldn't "sampling_params" be included in the curl request? I omitted it and I got this:

```
Traceback (most recent call last):
  File "/home/paperspace/.local/lib/python3.9/site-packages/bentoml/_internal/server/http_app.py", line 341, in api_func
    output = await api.func(*args)
  File "/home/paperspace/Documents/service.py", line 28, in generate
    n = request['sampling_params'].pop('n', 1)
KeyError: 'sampling_params'
```

**Review by Sherlock113 — COMMENTED** [review]
Minor tweaks. And when trying to send a request, I got some really strange response...Is this expected because of the model?

```
$ curl -X 'POST' \
>   'http://0.0.0.0:3000/v1/generate' \
>   -H 'accept: application/json' \
>   -H 'Content-Type: application/json' \
>   -d '{"prompt": "What is the meaning of life?", "stream": "False", "sampling_params": {"temperature": 0.73, "logprobs": 1}}'

hateningfetsuestionndinningmeomendinessotherslprsandllothersndoseutatelerheuththeningfefferentryonetsersonalotfinedyonetschvidualscoverningposefendilemingitsoediblyecauseeoveredningyoueywithoutdenyingtoonemoldoinsteadkingningfestartglfspywhatsledandtthievendentifeortstenggureeonemeaningisereel
```

File: docs/source/quickstarts/deploy-a-large-language-model-with-openllm-and-bentoml.rst [hunk 2]
```diff
@@ -85,30 +120,51 @@ The server is now active at `http://0.0.0.0:3000 <http://0.0.0.0:3000/>`_. You c
...
+               -H 'accept: application/json' \
+               -H 'Content-Type: application/json' \
+               -d '{"prompt": "What is the meaning of life?", "stream": "False"}'
```
Comment (aarnphm) [inline docs/source/quickstarts/deploy-a-large-language-model-with-openllm-and-bentoml.rst:132]:
```suggestion
               -d '{"prompt": "What is the meaning of life?", "stream": "False", "sampling_params": {"temperature": 0.73}}'
```

File: docs/source/quickstarts/deploy-a-large-language-model-with-openllm-and-bentoml.rst [hunk 3]
```diff
@@ -85,30 +120,51 @@ The server is now active at `http://0.0.0.0:3000 <http://0.0.0.0:3000/>`_. You c
...
+
+        For streaming generation
+
+        .. code-block:: bash
+
+           curl -X 'POST' -N \
+               'http://0.0.0.0:3000/v1/generate' \
+               -H 'accept: application/json' \
+               -H 'Content-Type: application/json' \
+               -d '{"prompt": "What is the meaning of life?", "stream": "True"}'
```
Comment (aarnphm) [inline docs/source/quickstarts/deploy-a-large-language-model-with-openllm-and-bentoml.rst:142]:
```suggestion
               -d '{"prompt": "What is the meaning of life?", "stream": "True", "sampling_params": {"temperature": 0.73}}'
```

File: docs/source/quickstarts/deploy-a-large-language-model-with-openllm-and-bentoml.rst (hunk 1 above)
Comment (aarnphm) [reply docs/source/quickstarts/deploy-a-large-language-model-with-openllm-and-bentoml.rst:129]: fixed.

**Comment by aarnphm** [comment]
@Sherlock113 PTAL

**Pull request merged**